src/packages.builds

```diff
@@ -16,6 +16,12 @@
     <Project Include="*\pkg\**\*.pkgproj" Condition="'$(BuildAllConfigurations)' == 'true' OR '$(DotNetBuildFromSource)' == 'true'">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
+     <Project Include="$(MSBuildThisFileDirectory)System.Data.SqlClient\pkg\System.Data.SqlClient.pkgproj">
```

Comment:
These changes were made in the wrong ItemGroup, they should have instead been made in the one bellow which is the one used for servicing.

https://github.com/dotnet/corefx/blob/c6f58cb8a0946b89c0faad564bbce8a102ca7211/src/packages.builds#L27-L35

The result is that these two packages are not going to be built as part of the official build, so in order for the packages to get serviced that will need to be fixed.